src/tests/utils/sample-url-generation.spec.tsx

```diff
@@ -80,14 +80,15 @@ describe('Sample Url Generation', () => {
 
 describe('hasWhiteSpaces should', () => {
   const invalidUrls = [
-    {url: ' https://graph.microsoft.com/v1.0/me', output: true},
+    {url: ' https://graph.microsoft.com/v1.0/me', output: false},
```

Comment:
*question*: Why should we allow the whitespace before the string?

Reply:
This was a suggestion by @millicentachieng. The leading spaces do not affect the url. You can still successfully run a query with leading spaces.

Reply:
ooh that's alright